src/test/kotlin/blackjack/GamblerTest.kt

```diff
@@ -0,0 +1,23 @@
+package blackjack
+
+import blackjack.model.*
```

Comment:
I recommend avoiding `wildcard`(`*`) in imports.
Can you guess the reason behind it? :)